### ISSUE TITLE
foobar crash after DUI-Installation

### ISSUE THREAD

**Issue by ASopH** [Chronial/foo_chronflow#1]
after installation in a dui environment foobar crashes. just one click at the chronflow-part of foobar
the attachment of the foobar crash-report fails. i post it in this message:

```
begin ---------------------------------------------------------------------------
Illegal operation:
Code: C0000005h, flags: 00000000h, address: 5B397845h
Access violation, operation: read, address: 00000000h

Call path not available.

Code bytes (5B397845h):
5B397805h:  00 00 00 8B D9 89 5D DC 8D 43 24 C6 45 F3 00 50
5B397815h:  89 45 D8 FF 15 10 22 3E 5B 8B 43 14 8D 4B 14 FF
5B397825h:  10 8B F8 33 F6 89 7D E0 89 75 E4 85 FF 0F 8E 9F
5B397835h:  02 00 00 8B 5D 0C EB 03 8D 49 00 8B 45 08 6A 00
5B397845h:  8B 08 8B 45 DC 8B 11 8B 40 18 8D 04 B0 50 53 6A
5B397855h:  00 FF 52 10 8B 03 8B CB FF 50 10 85 C0 0F 84 4B
5B397865h:  02 00 00 C7 45 C0 94 9C 3E 5B C7 45 C4 00 00 00
5B397875h:  00 C7 45 C8 00 00 00 00 C7 45 CC 00 00 00 00 C7

Stack (0F6FF848h):
0F6FF828h:  00000400 00000000 00000400 0F6FF850
0F6FF838h:  5B3880B0 0DA99320 0F6FF8AC 5B39781E
0F6FF848h:  00000000 1CC03EE9 0DE78C70 0DA99320
0F6FF858h:  0E0BB938 0DE78CC4 0F6FF940 0E0BD438
0F6FF868h:  0DE78D10 0F6FF8F4 5B384579 1CC03E3D
0F6FF878h:  5B384595 0F6FF88C 5B394DA8 0DA99344
0F6FF888h:  0DA99320 00000002 00000000 00000400
0F6FF898h:  0DA99320 000BB938 0F6FF8E0 5B3DE980
0F6FF8A8h:  FFFFFFFF 0F6FF8EC 5B397DD9 00000000
0F6FF8B8h:  0F6FF8C8 1CC03EA9 0CD32130 0DEB9570
0F6FF8C8h:  5B3EF364 0DDBC170 00000000 00000400
0F6FF8D8h:  00000000 00000000 0F6FF998 5B3DEA41
0F6FF8E8h:  00000001 0F6FF904 5B38391C 0F6FF92C
0F6FF8F8h:  0CD32130 0F6FF9A4 00000000 0F6FF9A4
0F6FF908h:  5B383680 0F6FF92C 0CD32130 1CC03FE1
0F6FF918h:  00000000 0DE78C70 FFFFFFFF 5B386A3A
0F6FF928h:  0DEB9570 0DEB9570 0DE78CC4 0DEB9110
0F6FF938h:  0DA99320 5B386601 0DE78D10 0DEB9501
0F6FF948h:  0DE78CC4 0E0BB938 0DE78CC4 0000005E
0F6FF958h:  0DE78C70 0000005E 0CD3218C 0DE78CD0

Registers:
EAX: 00000000, EBX: 0F6FF8C8, ECX: 0DA99334, EDX: FF717000
ESI: 00000000, EDI: 00000002, EBP: 0F6FF8AC, ESP: 0F6FF848

Crash location:
Module: foo_chronflow
Offset: 17845h

Loaded modules:
foobar2000                       loaded at 000D0000h - 0029D000h
ntdll                            loaded at 77800000h - 77979000h
KERNEL32                         loaded at 74F10000h - 75000000h
KERNELBASE                       loaded at 75030000h - 751A6000h
SHLWAPI                          loaded at 77410000h - 77454000h
msvcrt                           loaded at 771F0000h - 772AE000h
combase                          loaded at 75AF0000h - 75CAA000h
RPCRT4                           loaded at 752A0000h - 7534C000h
COMCTL32                         loaded at 6A7C0000h - 6A9C9000h
WINMM                            loaded at 6F3C0000h - 6F3E4000h
SspiCli                          loaded at 748F0000h - 7490E000h
GDI32                            loaded at 77460000h - 775AD000h
CRYPTBASE                        loaded at 748E0000h - 748EA000h
USER32                           loaded at 74A50000h - 74B90000h
bcryptPrimitives                 loaded at 74880000h - 748D9000h
sechost                          loaded at 75900000h - 75943000h
DSOUND                           loaded at 5C9B0000h - 5CA36000h
ADVAPI32                         loaded at 77720000h - 7779B000h
SHELL32                          loaded at 75DD0000h - 7718E000h
powrprof                         loaded at 772C0000h - 77304000h
windows.storage                  loaded at 75350000h - 7582C000h
WINMMBASE                        loaded at 6F390000h - 6F3B3000h
kernel.appcore                   loaded at 75CB0000h - 75CBC000h
shcore                           loaded at 74D50000h - 74DDD000h
cfgmgr32                         loaded at 74910000h - 74946000h
profapi                          loaded at 75AD0000h - 75ADF000h
ole32                            loaded at 751B0000h - 7529A000h
CRYPT32                          loaded at 75950000h - 75AC5000h
UxTheme                          loaded at 6A740000h - 6A7B5000h
MSASN1                           loaded at 75AE0000h - 75AEE000h
DEVOBJ                           loaded at 6EF00000h - 6EF21000h
zlib1                            loaded at 62E80000h - 62E9F000h
shared                           loaded at 5C2D0000h - 5C2FD000h
Secur32                          loaded at 74510000h - 7451A000h
gdiplus                          loaded at 6A9D0000h - 6AB3B000h
WINHTTP                          loaded at 71D40000h - 71DE7000h
imagehlp                         loaded at 75850000h - 75869000h
COMDLG32                         loaded at 75CC0000h - 75D7E000h
dbghelp                          loaded at 74650000h - 7478F000h
dbgcore                          loaded at 745A0000h - 745C1000h
IMM32                            loaded at 75000000h - 7502B000h
MSCTF                            loaded at 74DE0000h - 74F00000h
dwmapi                           loaded at 699E0000h - 699FD000h
foo_textdisplay                  loaded at 5C280000h - 5C2CB000h
foo_input_std                    loaded at 5C120000h - 5C27C000h
avcodec-fb2k-56                  loaded at 5BEB0000h - 5C08E000h
avutil-fb2k-54                   loaded at 5C090000h - 5C11A000h
MSACM32                          loaded at 5BE90000h - 5BEA8000h
foo_dsp_eq                       loaded at 5BE50000h - 5BE87000h
foo_facets                       loaded at 5BC90000h - 5BE45000h
OLEAUT32                         loaded at 77310000h - 773A2000h
MSIMG32                          loaded at 6F350000h - 6F356000h
foo_wave_seekbar                 loaded at 5BAE0000h - 5BC89000h
WS2_32                           loaded at 777A0000h - 777FC000h
NSI                              loaded at 772B0000h - 772B7000h
PSAPI                            loaded at 74F00000h - 74F06000h
MSVCR120                         loaded at 5B9F0000h - 5BADE000h
MSVCP120                         loaded at 5B970000h - 5B9E1000h
IPHLPAPI                         loaded at 74350000h - 74380000h
WINNSI                           loaded at 74340000h - 74348000h
mswsock                          loaded at 71830000h - 7187E000h
foo_simplaylist                  loaded at 5B820000h - 5B96C000h
foo_albumlist                    loaded at 5B7C0000h - 5B81B000h
foo_unpack                       loaded at 5B770000h - 5B7BC000h
foo_vis_shpeck                   loaded at 5B720000h - 5B761000h
foo_dsp_std                      loaded at 5B6E0000h - 5B717000h
foo_rgscan                       loaded at 5B680000h - 5B6DF000h
foo_freedb2                      loaded at 5B630000h - 5B680000h
foo_converter                    loaded at 5B5A0000h - 5B628000h
foo_ui_std                       loaded at 5B490000h - 5B59E000h
foo_cdda                         loaded at 5B430000h - 5B481000h
foo_chronflow                    loaded at 5B380000h - 5B422000h
OPENGL32                         loaded at 5B2A0000h - 5B380000h
glfw3                            loaded at 5B160000h - 5B16F000h
DDRAW                            loaded at 5B1B0000h - 5B29B000h
GLU32                            loaded at 5B180000h - 5B1A5000h
DCIMAN32                         loaded at 5B170000h - 5B177000h
foo_dynamic_range                loaded at 10000000h - 10068000h
foo_fileops                      loaded at 5B110000h - 5B15E000h
foo_uie_wsh_panel_mod            loaded at 5B020000h - 5B10A000h
foo_uie_lyrics3                  loaded at 03A50000h - 03B05000h
WININET                          loaded at 71120000h - 71344000h
foo_quicksearch                  loaded at 5AFC0000h - 5B017000h
WindowsCodecs                    loaded at 62F40000h - 630B1000h
bcrypt                           loaded at 74220000h - 7423B000h
frontend_direct2d                loaded at 5AF70000h - 5AFB1000h
d2d1                             loaded at 64B60000h - 64FE8000h
frontend_direct3d9               loaded at 5AF20000h - 5AF70000h
d3d9                             loaded at 5FC30000h - 5FDFC000h
d3dx9_42                         loaded at 5AD30000h - 5AF15000h
SciLexer                         loaded at 5AC80000h - 5AD22000h
clbcatq                          loaded at 75870000h - 758F2000h
MMDevApi                         loaded at 5EA70000h - 5EAC4000h
PROPSYS                          loaded at 71350000h - 71492000h
AUDIOSES                         loaded at 5DD10000h - 5DD78000h
wintypes                         loaded at 5DC40000h - 5DD05000h
cryptsp                          loaded at 73F20000h - 73F33000h
rsaenh                           loaded at 73600000h - 7362F000h
dataexchange                     loaded at 65010000h - 6504F000h
d3d11                            loaded at 648A0000h - 64AB3000h
dcomp                            loaded at 64AC0000h - 64B5C000h
dxgi                             loaded at 64760000h - 647DE000h
twinapi.appcore                  loaded at 646A0000h - 6475C000h
USERENV                          loaded at 74440000h - 74459000h
sxs                              loaded at 6E1D0000h - 6E250000h
jscript                          loaded at 5ABD0000h - 5AC73000h
amsi                             loaded at 5ABC0000h - 5ABCD000h
MpOav                            loaded at 5ABA0000h - 5ABB6000h
VERSION                          loaded at 74380000h - 74388000h
DWrite                           loaded at 65070000h - 65260000h
tiptsf                           loaded at 5AB20000h - 5AB9D000h
vis_milk2                        loaded at 08280000h - 08718000h
MSVCR90                          loaded at 74390000h - 74433000h
nvd3dum                          loaded at 5F020000h - 5FC23000h
d3dx9_36                         loaded at 5A770000h - 5AB19000h
usp10                            loaded at 75830000h - 75846000h
d3dx9_31                         loaded at 00400000h - 00667000h
edputil                          loaded at 5A740000h - 5A768000h
iertutil                         loaded at 71F60000h - 72221000h
policymanager                    loaded at 5A710000h - 5A73F000h
XmlLite                          loaded at 6EE50000h - 6EE7D000h
msvcp110_win                     loaded at 5A6A0000h - 5A705000h
nvoglv32                         loaded at 583F0000h - 595C8000h
SETUPAPI                         loaded at 74BA0000h - 74D45000h
WTSAPI32                         loaded at 74240000h - 7424F000h
WINTRUST                         loaded at 75D80000h - 75DC2000h
ntmarta                          loaded at 71C20000h - 71C48000h
WINSTA                           loaded at 71C70000h - 71CB4000h

Stack dump analysis:
Address: 5B384579h (foo_chronflow+4579h)
Address: 5B384595h (foo_chronflow+4595h)
Address: 5B394DA8h (foo_chronflow+14DA8h)
Address: 5B3DE980h (foo_chronflow+5E980h), symbol: "foobar2000_get_interface" (+32980h)
Address: 5B397DD9h (foo_chronflow+17DD9h)
Address: 5B3EF364h (foo_chronflow+6F364h), symbol: "foobar2000_get_interface" (+43364h)
Address: 5B3DEA41h (foo_chronflow+5EA41h), symbol: "foobar2000_get_interface" (+32A41h)
Address: 5B38391Ch (foo_chronflow+391Ch)
Address: 5B383680h (foo_chronflow+3680h)
Address: 5B386A3Ah (foo_chronflow+6A3Ah)
Address: 5B386601h (foo_chronflow+6601h)
Address: 5B3DD038h (foo_chronflow+5D038h), symbol: "foobar2000_get_interface" (+31038h)
Address: 5B382AF0h (foo_chronflow+2AF0h)
Address: 5B3B1628h (foo_chronflow+31628h), symbol: "foobar2000_get_interface" (+5628h)
Address: 5B3B993Ah (foo_chronflow+3993Ah), symbol: "foobar2000_get_interface" (+D93Ah)
Address: 5B3DCF70h (foo_chronflow+5CF70h), symbol: "foobar2000_get_interface" (+30F70h)
Address: 5B3822DBh (foo_chronflow+22DBh)
Address: 5B3B157Ch (foo_chronflow+3157Ch), symbol: "foobar2000_get_interface" (+557Ch)
Address: 5B3B1628h (foo_chronflow+31628h), symbol: "foobar2000_get_interface" (+5628h)
Address: 5B3B7E30h (foo_chronflow+37E30h), symbol: "foobar2000_get_interface" (+BE30h)
Address: 5B3B16A4h (foo_chronflow+316A4h), symbol: "foobar2000_get_interface" (+56A4h)
Address: 5B3B1628h (foo_chronflow+31628h), symbol: "foobar2000_get_interface" (+5628h)
Address: 74F23744h (KERNEL32+13744h), symbol: "BaseThreadInitThunk" (+24h)
Address: 74F23720h (KERNEL32+13720h), symbol: "BaseThreadInitThunk" (+0h)
Address: 7785A064h (ntdll+5A064h), symbol: "RtlSetCurrentTransaction" (+D4h)
Address: 750F8CD0h (KERNELBASE+C8CD0h), symbol: "UnhandledExceptionFilter" (+0h)
Address: 77870D40h (ntdll+70D40h), symbol: "wcstombs" (+90h)
Address: 7785A02Fh (ntdll+5A02Fh), symbol: "RtlSetCurrentTransaction" (+9Fh)
Address: 7787D7B4h (ntdll+7D7B4h), symbol: "RtlCaptureContext" (+D4h)
Address: 5B3B1628h (foo_chronflow+31628h), symbol: "foobar2000_get_interface" (+5628h)
Address: 000D000Eh (foobar2000+Eh)
Address: 000E000Fh (foobar2000+1000Fh)
Address: 000F0010h (foobar2000+20010h)
Address: 00100011h (foobar2000+30011h)
Address: 00110012h (foobar2000+40012h)
Address: 00120013h (foobar2000+50013h)
Address: 00130014h (foobar2000+60014h)
Address: 00140015h (foobar2000+70015h)
Address: 00150016h (foobar2000+80016h)
Address: 00160017h (foobar2000+90017h)
Address: 00170018h (foobar2000+A0018h)
Address: 00180019h (foobar2000+B0019h)
Address: 0019001Ah (foobar2000+C001Ah)
Address: 001A001Bh (foobar2000+D001Bh)
Address: 001B001Ch (foobar2000+E001Ch)
Address: 001C001Dh (foobar2000+F001Dh)
Address: 001D001Eh (foobar2000+10001Eh)
Address: 001E001Fh (foobar2000+11001Fh)
Address: 001F0020h (foobar2000+120020h)
Address: 00200021h (foobar2000+130021h)
Address: 00210023h (foobar2000+140023h)
Address: 00220025h (foobar2000+150025h)
Address: 00230027h (foobar2000+160027h)
Address: 00240029h (foobar2000+170029h)
Address: 0025002Bh (foobar2000+18002Bh)
Address: 0026002Dh (foobar2000+19002Dh)
Address: 0027002Fh (foobar2000+1A002Fh)
Address: 00280031h (foobar2000+1B0031h)
Address: 00290033h (foobar2000+1C0033h)
Address: 00400081h (d3dx9_31+81h)
Address: 00410089h (d3dx9_31+10089h)
Address: 00420091h (d3dx9_31+20091h)
Address: 00430099h (d3dx9_31+30099h), symbol: "D3DXDebugMute" (+448Bh)
Address: 004400A1h (d3dx9_31+400A1h), symbol: "D3DXFrameCalculateBoundingSphere" (+ED5h)
Address: 004500A9h (d3dx9_31+500A9h), symbol: "D3DXGetDeclLength" (+165Fh)
Address: 004600B1h (d3dx9_31+600B1h), symbol: "D3DXSplitMesh" (+D77h)
Address: 004700B9h (d3dx9_31+700B9h), symbol: "D3DXSimplifyMesh" (+551Fh)
Address: 004800C1h (d3dx9_31+800C1h), symbol: "D3DXTriPatchSize" (+4E42h)
Address: 004900C9h (d3dx9_31+900C9h), symbol: "D3DXTriPatchSize" (+14E4Ah)
Address: 004A00D1h (d3dx9_31+A00D1h), symbol: "D3DXTriPatchSize" (+24E52h)
Address: 004B00D9h (d3dx9_31+B00D9h), symbol: "D3DXTriPatchSize" (+34E5Ah)
Address: 004C00E1h (d3dx9_31+C00E1h), symbol: "D3DXFillVolumeTextureTX" (+48h)
Address: 004D00E9h (d3dx9_31+D00E9h), symbol: "D3DXSHProjectCubeMap" (+D499h)
Address: 004E00F1h (d3dx9_31+E00F1h), symbol: "D3DXGatherFragments" (+2B5Eh)
Address: 004F00F9h (d3dx9_31+F00F9h), symbol: "D3DXCreateEffect" (+605Bh)
Address: 00500101h (d3dx9_31+100101h), symbol: "D3DXCreateEffect" (+16063h)
Address: 00510111h (d3dx9_31+110111h), symbol: "D3DXCreateEffect" (+26073h)
Address: 00520121h (d3dx9_31+120121h), symbol: "D3DXCreateEffect" (+36083h)
Address: 00530131h (d3dx9_31+130131h), symbol: "D3DXCreateEffect" (+46093h)
Address: 00540141h (d3dx9_31+140141h), symbol: "D3DXCreateEffect" (+560A3h)
Address: 00550151h (d3dx9_31+150151h), symbol: "D3DXCreateEffect" (+660B3h)
Address: 00560161h (d3dx9_31+160161h), symbol: "D3DXCreateFragmentLinker" (+6152h)
Address: 00570171h (d3dx9_31+170171h), symbol: "D3DXCreateFragmentLinker" (+16162h)
Address: 00580181h (d3dx9_31+180181h), symbol: "D3DXCreatePRTEngine" (+1DF5h)
Address: 00590191h (d3dx9_31+190191h), symbol: "D3DXSHPRTCompSplitMeshSC" (+9F00h)
Address: 005A01A1h (d3dx9_31+1A01A1h), symbol: "D3DXSHPRTCompSplitMeshSC" (+19F10h)
Address: 005B01B1h (d3dx9_31+1B01B1h), symbol: "D3DXFileCreate" (+896Fh)
Address: 005C01C1h (d3dx9_31+1C01C1h), symbol: "D3DXFileCreate" (+1897Fh)
Address: 005D01D1h (d3dx9_31+1D01D1h), symbol: "D3DXFileCreate" (+2898Fh)
Address: 005E01E1h (d3dx9_31+1E01E1h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+567Ah)
Address: 005F01F1h (d3dx9_31+1F01F1h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+1568Ah)
Address: 00600201h (d3dx9_31+200201h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+2569Ah)
Address: 00610221h (d3dx9_31+210221h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+356BAh)
Address: 00620241h (d3dx9_31+220241h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+456DAh)
Address: 00630261h (d3dx9_31+230261h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+556FAh)
Address: 00640281h (d3dx9_31+240281h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+6571Ah)
Address: 006502A1h (d3dx9_31+2502A1h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+7573Ah)
Address: 006602C1h (d3dx9_31+2602C1h), symbol: "D3DXComputeIMTFromPerTexelSignal" (+8575Ah)

Environment:
App: foobar2000 v1.3.8
UI: Default User Interface 0.9.5

Components:
Core (2015-03-26 07:45:52 UTC)
    foobar2000 core 1.3.8
foo_albumlist.dll (2015-03-26 07:42:40 UTC)
    Album List 4.5
foo_cdda.dll (2015-03-26 07:42:18 UTC)
    CD Audio Decoder 3.0
foo_chronflow.dll (2015-09-27 06:35:21 UTC)
    Chronial's Coverflow 0.4.3
foo_converter.dll (2015-03-26 07:42:24 UTC)
    Converter 1.5
foo_dsp_eq.dll (2015-03-07 10:19:44 UTC)
    Equalizer 1.0
foo_dsp_std.dll (2015-03-26 07:42:26 UTC)
    Standard DSP Array 1.3.1
foo_dynamic_range.dll (2011-11-12 22:18:36 UTC)
    Dynamic Range Meter 1.1.1
foo_facets.dll (2011-08-18 18:06:16 UTC)
    Facets 1.0
foo_fileops.dll (2015-03-07 10:19:10 UTC)
    File Operations 2.2.1
foo_freedb2.dll (2015-03-07 10:19:36 UTC)
    Online Tagger 0.7
foo_input_std.dll (2015-03-26 07:45:52 UTC)
    Standard Input Array 1.0
foo_quicksearch.dll (2012-10-29 15:44:48 UTC)
    Quick Search Toolbar 3.4
foo_rgscan.dll (2015-03-26 07:43:12 UTC)
    ReplayGain Scanner 2.2.1
foo_simplaylist.dll (2011-08-18 18:01:36 UTC)
    SimPlaylist 1.0
foo_textdisplay.dll (2011-05-29 14:03:58 UTC)
    Text Display UI Element 1.1 beta 1
foo_ui_std.dll (2015-03-26 07:45:52 UTC)
    Default User Interface 0.9.5
foo_uie_lyrics3.dll (2015-02-16 17:26:16 UTC)
    Lyric Show Panel 3 0.4.5
foo_uie_wsh_panel_mod.dll (2015-08-17 15:55:33 UTC)
    WSH Panel Mod 1.5.10
foo_unpack.dll (2015-03-26 07:42:42 UTC)
    ZIP/GZIP/RAR Reader 1.7.1
foo_vis_shpeck.dll (2009-09-28 09:32:16 UTC)
    Shpeck - Winamp vis plugins wrapper 0.3.7
foo_wave_seekbar.dll (2015-09-11 20:48:42 UTC)
    Waveform seekbar 0.2.45

Recent events:
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_dsp_std.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_rgscan.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_freedb2.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_converter.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_ui_std.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_cdda.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\user-components\foo_chronflow\foo_chronflow.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\user-components\foo_dynamic_range\foo_dynamic_range.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\components\foo_fileops.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\user-components\foo_uie_wsh_panel_mod\foo_uie_wsh_panel_mod.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\user-components\foo_uie_lyrics3\foo_uie_lyrics3.dll
Loading DLL: C:\Users\..\Desktop\foobar2000 v1.3.8 DUI-Test\user-components\foo_quicksearch\foo_quicksearch.dll
Components loaded in: 0:00.044046
Pre config read
Configuration read in: 0:00.003791
Starting playlist loading
Watching: E:\
WSH Panel Mod ({20EE7991-F61B-408E-8D3F-EE7F82700286}): initialized in 5 ms
WSH Panel Mod ({9B9E2CF6-6B13-41CF-ADB4-0BFF45643620}): initialized in 1 ms
User Interface initialized in: 0:00.229872
Processing commandline
initquit::init entry
initquit::init exit
Startup time : 0:00.398261
[Shpeck] Unsupported WM_WA_IPC 1336 (param = 0)
[Shpeck] Unsupported WM_WA_IPC 1335 (param = 0)
Seekbar: taking Direct3D9 path.
Seekbar: Frontend initialized.
[Shpeck] Unsupported WM_WA_IPC 3044 (param = 0)
foo_chronflow Collection Generation: 0:00.002751


Machine specifications:
OS: Windows 6.3.9600 x64
CPU: Intel(R) Core(TM) i5-3570 CPU @ 3.40GHz, features: MMX SSE SSE2 SSE3 SSE4.1 SSE4.2
CPU threads: 4
Audio: Lautsprecher (Realtek High Definition Audio); Realtek Digital Output (Realtek High Definition Audio)
end  ---------------------------------------------------------------------------
```

**Comment by Chronial** [comment]
Could you send me the dmp file for this crash, pls? You can find it in `%AppData%\foobar2000\crash reports`.

**Comment by ASopH** [comment]
the upload fails every time... :-(

**Comment by Chronial** [comment]
You can send it to my email address: ![mail](http://chronial.de/foobar2000/mail.gif)

**Comment by ASopH** [comment]
done.. thx

**Comment by Chronial** [comment]
Thanks a lot for your support. I think I found the bug and it should be fixed by 88edbe05322df6aa64b3b460a9611f15500aca20.
